draft-packages/title-block-zen/KaizenDraft/TitleBlockZen/TitleBlockZen.tsx

```diff
@@ -27,20 +37,47 @@ type Props = {
   sectionTitle?: string
   sectionTitleDescription?: string
   handleHamburgerClick?: (event: React.MouseEvent) => void
-  primaryActions?: PrimaryActions
-  secondaryActions?: SecondaryActions
+  primaryAction: PrimaryActionProps
```

Comment:
We may need to make the `primaryAction` and optional thing as some pages have no calls to action

Reply:
Ah good catch. I'll do that now

Reply:
✅